src/metabase/pulse/render/js_svg.clj

```diff
@@ -30,16 +31,19 @@
     (js/load-resource bundle-path)
     (js/load-resource interface-path)))
 
-(defn- static-viz-context
-  "Load the static viz js bundle into a new graal js context."
-  []
-  (load-viz-bundle (js/context)))
-
-(def ^:private ^Context context
-  "Javascript context suitable for evaluating the charts. It has the chart bundle and the above `src-api` in its
-  environment suitable for creating charts."
+(def ^:private ^Context static-viz-context-delay
```

Comment:
Does this type hint really help? Shouldn't it be on `context` below?

Reply:
Yep, good catch